javascript/src/seedsource-ui/containers/Variable.tsx

```diff
@@ -229,29 +230,31 @@ const Variable = (props: ConnectedProps<typeof connector>) => {
             onToggle()
           }}
         />
-
-        <ReactTooltip id={`${name}_Tooltip`} className="variable-tooltip" place="right" effect="solid">
-          <h5 className="title is-5 margin-bottom-5">
-            {name}: {label}
-          </h5>
-          <div>
-            <span className="tooltip-label">{customMode ? t`Custom value:` : t`Value at point:`}</span>{' '}
-            <strong>{value}</strong>
-          </div>
-          <div>
-            <span className="tooltip-label">{t`Transfer limit (+/-):`}</span>
-            <strong>
-              {transfer} {units[unit].label} {transferIsModified ? `(${t`modified`})` : ''}
-            </strong>
-          </div>
-          <div>
-            <span className="tooltip-label">{t`Avg.transfer limit for zone set:`}</span>
-            <strong>
-              {avgTransfer} {units[unit].label}
-            </strong>
-          </div>
-          {climaticCenter}
-        </ReactTooltip>
+        {createPortal(
```

Comment:
ReactTooltip doesn't create its own portal?

Reply:
I'm not sure. I didn't really question this code from Bill, I'll look into it.

Reply:
They weren't necessary

Reply:
Thanks for checking it out.